portability-worker/src/main/java/org/dataportabilityproject/worker/WorkerExtensionContext.java

```diff
@@ -30,16 +33,38 @@
  * {@link ExtensionContext} used by the worker.
  */
 final class WorkerExtensionContext implements ExtensionContext {
+  // TODO: build script, which should take cloud extension name and environment (e.g. local) as
```

Comment:
Does it make sense to move this TODO to the build script?

Reply:
yes, good point